tests/test_a11y.py

```diff
@@ -291,3 +291,34 @@ def test_breadcrumb_expansion(page: Page, url_base: str) -> None:
     expect(page.get_by_label("Breadcrumb").get_by_role("list")).to_contain_text(
         "Update Sphinx configuration during the build"
     )
+
+
+@pytest.mark.a11y
+def test_search_as_you_type(page: Page, url_base: str) -> None:
```

Comment:
@drammock re: the search results container itself receiving focus, it seems like this test should have failed? Does CI not run the a11y tests on different OS / browser permutations?

Reply:
yeah, seems it should have 🤷🏻 
CI only runs a11y tests in one job (ubuntu-latest, py3.12).

Reply:
Added #2095 to track the issue. I assume we'll want to address that in a separate PR